examples/go/aws/go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/hashicorp/terraform-cdk/examples/go/aws
+
+go 1.16
+
+require (
+	github.com/aws/constructs-go/constructs/v3 v3.3.75
+	github.com/aws/jsii-runtime-go v1.29.0
+	github.com/hashicorp/terraform-cdk-go/cdktf v0.0.0
+	github.com/hashicorp/terraform-cdk-provider-hashicorp-aws-go/aws v0.0.0
+)
+
+// use for local testing
+// replace github.com/hashicorp/terraform-cdk-go/cdktf => ../../../packages/cdktf/dist/go/cdktf
```

Comment:
Is this still relevant? If so, this probably also default to `dist`?

Reply:
Updated the comment. It is relevant for the examples to run against the locally built version.

fyi: When using the template via `cdktf init`, it'll use the supplied dist directory (if any).